lxd/containers_post.go

```diff
@@ -14,12 +14,57 @@ import (
 	"github.com/lxc/lxd/shared"
 )
 
-func btrfsCopyImage(hash string, name string, d *Daemon) error {
-	dpath := shared.VarPath("lxc", name)
-	imagefile := shared.VarPath("images", hash)
-	subvol := fmt.Sprintf("%s.btrfs", imagefile)
+func btrfsCmdIsInstalled() bool {
+	/*
+	 * Returns true if the "btrfs" tool is in PATH else false.
+	 *
+	 * TODO: Move this to the main code somewhere and call it once?
+	 */
+	out, err := exec.Command("which", "btrfs").Output()
```

Comment:
`exec.LookPath` may be more idiomatic here; that's what we do for the required programs in main.go.